scripts/store.js

```diff
@@ -220,18 +220,18 @@ async function addTemplate({ label }) {
     return template;
 }
 
-async function getTemplate({ id }, imageFields = "*") {
+async function getTemplate({ id }, withImages = true) {
     const templateRow = await knex
         .select("*")
         .from("template")
         .where({ id })
         .first();
 
-    if (imageFields === "none") {
+    if (!withImages) {
```

Comment:
I think some bugs came from this fields argument. Now it is boolean and not sent forward to getTemplateImages.